.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 10
```

Comment:
I ended up [reverting](https://github.com/TrueLayer/reqwest-middleware/pull/138) your PR on `reqwest-middleware` because the cargo upgrades were far too noisy. Dependabot was opening PRs for minor upgrades which is unhelpful since, for libraries, we aim closer to the minimum compatible version, rather than the latest one. For major version upgrades, I prefer to gauge demand by waiting for our users to open PRs/issues.

```suggestion
```

Reply:
Okay, thanks for reply